desk/app/groups.hoon

```diff
@@ -1451,7 +1451,7 @@
       go-core
     ::
         %edit
-      =.  cabals.group
+      =?  cabals.group  (~(has by cabals.group) sect)
         %+  ~(jab by cabals.group)  sect
         |=  cabal:g
         +<(meta meta.diff)
```

Comment:
You kind of want something like the following, except the `+go-fact-init` does too many things right now, so kind of hard to flow into that as-is without surprising side-effects.

```suggestion
      ?.  (~(has by cabals.group) sect)
        ::  we have incomplete data, resubscribe to bring us back in sync
        ::
        (go-sub & blah)
      =.  cabals.group
        %+  ~(jab by cabals.group)  sect
        |=  cabal:g
        +<(meta meta.diff)
```

In any case, this is just a crash that says "we're not in sync", and so we should try to get in sync somehow. Sweeping the problem under the rug may be fine, but I'll want to see a very big `::TODO` here and probably a matching linear ticket (or sub-issue on the groups refactor one).

Reply:
None of the above solves the root issue of _becoming_ out of sync, btw, but we don't have much to go on for that rn.